acceptance/allocator_terraform/main.tf

```diff
@@ -13,6 +13,7 @@ resource "google_compute_instance" "cockroach" {
   disk {
     image = "${var.gce_image}"
     size = "${var.cockroach_disk_size}" # GB
+    # type    = "pd-ssd" # uncomment for persistent SSD
```

Comment:
Ok, done.

On Tue, Jul 19, 2016 at 9:23 AM Peter Mattis <notifications@github.com>
wrote:

> In acceptance/allocator_terraform/main.tf
> <https://github.com/cockroachdb/cockroach/pull/7909#discussion_r71336286>:
>
> > @@ -13,6 +13,7 @@ resource "google_compute_instance" "cockroach" {
> >    disk {
> >      image = "${var.gce_image}"
> >      size = "${var.cockroach_disk_size}" # GB
> > +    # type    = "pd-ssd" # uncomment for persistent SSD
>
> No strong reason. I'm just imagining others will want to run the test with
> pd-ssd disks.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7909/files/2f1fa4ba848d593fb1998bd7f98e0cf34390fd05#r71336286>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE135ExWwH-pc9ZwrE1oqbbLvffc3Aq4ks5qXM-9gaJpZM4JPnbk>
> .
>
-- 

-- Tobias